playground/mocks/data/idp/idx/error-authenticator-enroll-phone-invalid-number.json

```diff
@@ -0,0 +1,209 @@
+                "label":"Okta Verify",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut23c6ZxGsdVvWOm0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"channel",
+                        "type":"string",
+                        "required":false,
+                        "options":[
+                          {
+                            "label":"QRCODE",
+                            "value":"qrcode"
+                          },
+                          {
+                            "label":"EMAIL",
+                            "value":"email"
+                          },
+                          {
+                            "label":"SMS",
+                            "value":"sms"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticators.value[0]"
+              },
+              {
+                "label":"Phone",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut23bgproCCEkFom0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"methodType",
+                        "type":"string",
+                        "required":false,
+                        "options":[
+                          {
+                            "label":"SMS",
+                            "value":"sms"
+                          },
+                          {
+                            "label":"Voice call",
+                            "value":"voice"
+                          }
+                        ]
+                      },
+                      {
+                        "name":"phoneNumber",
+                        "label":"Phone number",
+                        "required":false
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticators.value[1]"
+              }
+            ]
+          },
+          {
+            "name":"stateHandle",
+            "required":true,
+            "value":"02A2CNDXsH6luRQV4R-GOWt2_ynSdb3UWg-3Ce_-sN",
+            "visible":false,
+            "mutable":false
+          }
+        ],
+        "accepts":"application/json; okta-version=1.0.0"
+      }
+    ]
+  },
+  "messages":{
+    "type":"array",
+    "value":[
+      {
+        "message":"Invalid Phone Number.",
+        "i18n": {
+          "key": "api.factors.error.sms.invalid_phone"
+        },
+        "class":"ERROR"
+      }
+    ]
+  },
```

Comment:
@hassanmahmood-okta to confirm that the `messages` object will look like this after his PR for https://oktainc.atlassian.net/browse/OKTA-391131

Reply:
> @hassanmahmood-okta to confirm that the `messages` object will look like this after his PR for https://oktainc.atlassian.net/browse/OKTA-391131

Yes i confirm that the response will contain the message object as you have stated here.
```
"messages":{
    "type":"array",
    "value":[
      {
        "message":"Invalid Phone Number.",
        "i18n": {
          "key": "api.factors.error.sms.invalid_phone"
        },
        "class":"ERROR"
      }
    ]
  }
```